Modules/_sqlite/module.c

```diff
@@ -320,6 +320,115 @@ static const struct {
 #if SQLITE_VERSION_NUMBER >= 3007017
     DECLARE_ERROR_CODE(SQLITE_NOTICE),
     DECLARE_ERROR_CODE(SQLITE_WARNING),
+#endif
+    // Extended result code list
```

Comment:
Can you add here some link to the docs so we can check in the future more codes and the version constraints?

Reply:
Will do!

Reply:
BTW, the version constraints were a bit tricky; many of them were not mentioned in the SQLite changelog, so I used a simple shell script for grepping through all the SQLite versions, finding out when the various macros appeared.

Reply:
> so I used a simple shell script for grepping through all the SQLite versions

Then isn't it better to use  preprocessor checks to see if they are defined?

Reply:
That's also an option. I thought about that, but I though this was a little bit more elegant, IMO. Let me know if you want me to change it.

Reply:
I think modifying/adding a macro that adds the preprocessor checks + the addition may be a bit more future-proof.

Reply:
My main worry is how this can be extended in the future and what can happen if we do it incorrectly. With the checks, there is no possibility of getting if wrong :)

Reply:
True that! I'll fix it tomorrow :)

Reply:
_But_, if we keep the version checks as they are, it's easier to declutter the code when we up the SQLite version requirements.

Reply:
That's true. At this point, I trust your judgement. Whatever you think is best is going to be good enough :)

Just make sure to add enough comments so the next person knows how to add more stuff in the future

Reply:
All right; I'll think about it thrice before deciding :)

Reply:
I'm keeping the version checks for future decluttering. I added some comments, but I'm not satisfied with them (yet). Finding out when a new result code was added will probably involve some detective work anyway, given how the SQLite writes their changelogs.

Reply:
I think 26517fc8e692304c1c77a26fcbb9e6ca78989735 is ok. PTAL.